common/config/rush/pnpm-lock.yaml

```diff
@@ -1428,7 +1428,7 @@ importers:
       '@aws/workbench-core-authentication': link:../../authentication
       '@aws/workbench-core-authorization': link:../../authorization
       '@aws/workbench-core-base': link:../../base
-      '@aws/workbench-core-datasets': link:../../dataSets
+      '@aws/workbench-core-datasets': link:../../datasets
```

Comment:
This was my fault. WebStorm did a replacement of all the dynamic uses of `datasets` instead of just changing at the file I was using. I thought i cleaned everything up but guess I missed some.

Reply:
I figured it was a bulk rename issue